cms/djangoapps/contentstore/views/course.py

```diff
@@ -542,9 +542,9 @@ def _deprecated_blocks_info(course_module, deprecated_block_types):
     """
     data = {
         'block_types': deprecated_block_types,
```

Comment:
I don't think we need to pass `block_types` to the template anymore.  If not, maybe we can rename `block_types_enabled` to `block_types` (for brevity with no loss of clarity).